### DISCUSSION TITLE
سیستمی برای اتحاد و سازماندهی اعتصابات سراسری

### DISCUSSION THREAD

**Discussion by forwomanlifefreedom** [WeAreMahsaAmini/Core#491] [Ideas]
سلام دوستان

ایده کلی اینه که سیستمی بنویسیم که افرادی که قصد اعتصاب یا هماهنگی برای انجام کاری در جهت مبارزه در بک بخش، سازمان و... دارن، بتونن براحتی با هم هماهنگ بشن.

اول اینکه فرض براینه که دلایل اصلی ای که افراد اعتصاب نمیکنن خصوصا در بخشهای مهم مثل پالایشگاه ها  و مراکز مهم نفت و گاز اینه:

1) ترس از دستگیری و سرکوب یا مواجهه با اتهام لیدری اعتصابات درصورت اقدام برای سازماندهی افراد برای اعتصاب
2) ترس از اخراج شدن و ازدست دادن شغل و درامد
3) عدم اعتماد به همکاران برای انجام هماهنگی درصورت وجود جو امنیتی در اون سازمان
4) تصور بر عدم تاثیرگذار بودن، مثل: این همه مغازه بازن من یدونه اعتصاب کنم هیچ تاثیری روی بازار نمیذاره! 
5) نداشتن پس انداز و بدهکاری مالی 

این سیستم باید به این شکل باشه که هر فردی که قصد اعتصاب داره، به تنهایی وارد سیستم بشه و اون سازمان، بخش یا صنفی که در اون مشغول بکاره رو انتخاب کنه و مدارک لازم برای احراز هویتش که مشخص کنه این فرد حقیقی از کارکنان اون بخش یا صنفه رو ارایه بده و بعدش گزینه اعتصاب یا هر نوع شیوه دیگر مبارزه رو انتخاب کنه (هرزمانیکه خواست هم میتونه گزینه انصرافو بزنه).
 در طرف مقابل سیستم باید تعداد و اسم کارکنان مربوط به اون بخش رو بدونه که بدست اوردن این اطلاعات هم کار چندان سختی نیست.
در مرحله بعد سیستم با مطابقت اسم و مدارک اون فرد احراز هویت رو انجام میده و یک نفر رو به تعداد افرادیکه قصد اعتصاب در اون بخش رو دارن اضافه میکنه.
 اما چیزی که سایر کارکنان میبینن فقط تعداد افراده و هویت افراد شرکت کننده برای همدیگه یا سایر افراد ناشناس میمونه.
مثلا چیزی که کارکنان میبیند اینه: از 2000 کارگر 1000 نفر قصد اعتصاب دارن. که خود این موضوع هم به بقیه برای پیوستن به اعتصابات انگیزه میده. درنهایت وقتی تعداد به میزان قابل توجهی رسید (مثلا 1700 از 2000 نفر) سیستم اعلام اعتصاب میکنه و همه همزمان در یک زمان مشخص دست از کار میکشن یا دست به یک اقدام جمعی میزنن که از قبل در سیستم مشخص کردن.

همچنین سیستم در این صورت براحتی میتونه به افراد کمک مالی هم برسونه.

از این سیستم همه افراد با هدف اعتصاب و اعتراض سراسری میتونن استفاده کنن اعم از کارگران، کارمندان، بازاریان، هنرمندان، دانشجویان، دانش اموزان و... و حتی ارگان های نظامی مثل ارتش که فضای امنیتی شدیدی برای جلوگیری از شورش در اونجا حاکمه. 

دراینصورت تقریبا تمام موانعی که در بالا گفته شد که از ترس از تنها یا کم تعداد بودن یا مشکل بودن سازماندهی تعداد زیاد افراد و همزمانی برای انجام کار واحد ناشی میشد، از بین میره.

اما چالش بزرگ و اصلی تضمین امنیت اطلاعات افراد و جلب اعتماد افراد برای ورود به سیستم و احراز هویت و درکل اعتماد به سیستمه.
که برای جلب اعتماد احتمالا بشه از فعالان سیاسی که مورد قبول اکثریت مردم هستن، خواست که سیستم رو پروموت و تبلیغ کنن.

  درنهایت هدف اینه که با این سیستم بشه یک کار واحد و با تعداد زیاد افراد و بطور همزمان و پیوسته و بدون نیاز به لیدر و رهبر سازماندهی شه، درغیر اینصورت به دستگیری و سرکوب منتهی خواهد شد که همونطور که میدونید 200 نفر از کارگران پروژه ای که اعتصاب کرده بودند دستگیر شدند و درنتیجه اعتصابات اون بخش سرکوب شد.

 چون کین درد مشترک هرگز جدا جدا درمان نمیشود.

**Comment by IJustWantSomeFreedom** [comment]
ایده جالب و تقریبا کاملیه.

ولی فکر نکنم که نیازی به احراز هویت باشه، اگر کمک مالی نخوایم کنیم.
اگر نگران ترول ها هستید هم میتونیم یه محدودیت بزاریم که هر حساب کاربری میتونه حداکثر در یکی از شرکت ها و صنف های نامبرده اعتصاب کنه. و همچنین یه سری محدودیت دیگه مثل ip, captcha و ... که کار ترول و اسپمر ها هم سختتر بشه. هرچند، فکر نکنم کسی بخواد ترول کنه. آخه اگه خود بسیجی ها هم بیان اسپم کنن و الکی آمارو ببرن بالا بیشتر به ضرر خودشون تموم میشه.

به هر حال حتی اگر باز هم احتمال به هم زدن آمار هست، میتونیم به جای مشخصات شخصی فرد، مشخصاتی بگیریم که نشون دهده حقیقی بودن فرد باشه ولی شخصی نباشه. مثلا یه فرم نسبتا طولانی بدیم از مشخصات شرکت و یه سری سوال captcha که نه ربات ها توانش رو داشته باشن و نه بسیجی ها وقت این رو داشته باشن که دونه دونه همه سوال هارو جواب بدن.
همچنین جواب سوال ها هم توسط ادمین های پلتفرم چک میشه و مطمئن میشن که جواب ها واقعی بوده و صرفا فقط برای پر کردن فیلد ها نیست

**Reply by forwomanlifefreedom** [reply]
بنظرم شاید در نگاه اول ترول‌ها نخوان آمارو الکی بالا ببرن ولی همین که بقیه ادما بدونن که هرکسی و هر چند بار میتونه حساب باز کنه و بگه میخوام اعتصاب کنم، همین باعث بی‌اعتمادی بشه و ترول‌ها دقیقا بخاطر همین موضوع هم که شده بیان و آمارو خراب کنن و اصولا وقتی احراز هویتی درکار باشه همین باعث میشه کسایی که واقعا میخوان اعتصاب کنن وارد سیستم بشن.
و اون قسمت فرم سوالا هم من چیزی به ذهنم نمیاد که باعث تمایز بشه و اگرم باشه اینجوری باید برای تک به تک شرکتا و سازمانا فرم ساخته بشه و برای صنف های دیگه مثل بازاریا و... همچین چیزی نمیشه ساخت. البته نمیدونم شاید بشه، در این مورد اطلاعاتی ندارم و چیزی به ذهنم نمیرسه.
